clang/docs/UsersManual.rst

```diff
@@ -4401,13 +4413,18 @@ Execute ``clang-cl /?`` to see a list of supported options:
                               Instrument only functions from files where names don't match all the regexes separated by a semi-colon
       -fprofile-filter-files=<value>
                               Instrument only functions from files where names match any regex separated by a semi-colon
+      -fprofile-generate[=<dirname>]
```

Comment:
Rather than `<dirname>`, I think it'd be better to say `<pattern>` and point to https://clang.llvm.org/docs/UsersManual.html#profiling-with-instrumentation for more details.

Reply:
Updated description. It is actually a dirname, not pattern.

Reply:
This is really surprising but I just tested this and you're correct. Do you know why that's the case, that is why `-fprofile-generate=<dirname>` and `-fprofile-instr-generate=<filename_pattern>` have a different behavior?

Reply:
-fprofile-generate=  in GCC specifies a path which is a directory name (the reason is that GCC's profile data is dumped per module, so there is no single profile name).

When we introduced this option in Clang/LLVM, we just matched that behavior.